pom.xml

```diff
@@ -423,6 +423,10 @@
 					<artifactId>bcprov-jdk16</artifactId>
 					<groupId>bouncycastle.bcprov-jdk16</groupId>
```

Comment:
I'm pretty sure this exclusion is not needed, as orca.core is not providing this artifact.  This should probably be deleted.